snippets/base/templates/base/includes/snippet_js.html

```diff
@@ -411,6 +412,21 @@
         }
     }
 
+    // Bug 1238092. Got fixed in Fx45. We can drop this when Fx44
+    // usage drops significantly.
+    function keypressIsMine(snippet) {
+        function onKeyPress(e) {
+            e.stopPropagation();
+        }
+        function attachHandler(matches) {
+            for (var i = 0, len = matches.length; i < len; i++) {
+                matches[i].addEventListener('keypress', onKeyPress);
```

Comment:
probably not such a big deal but, if it is possible that there might be a few input and select elements, it can end up attaching a lot of event handlers. If this is not the case and the number is low 3-4 perhaps, then not to worry.

If it can get higher, then we should perhaps rather attach an event handler to the parent element and, if the `event.target` of the `keypress` event was an `input` or `select` we call `onKeyPress`, else we just let it pass through.

Something like:

```
parent.addEventListener('keypress', function(event) {
    var tagName = event.target.tagName;

    if (tagName === "INPUT" || tagName === 'SELECT') {
        onKeyPress();
    }
});
```